documentation/source/newsfragments/2294.change.rst

```diff
@@ -0,0 +1 @@
+VHDL allows signals to have a length of zero bit. BinaryValue failed when trying to convert a empty string into an integer. Now a vector of length zero reads as 0.
```

Comment:
I think this should be `.fix` or `.bug`, I forget what the filename convention is.

Also:
```suggestion
VHDL signals that are zero bits in width now read as integer zero, instead of raising an exception.
```

Reply:
According to the README we have the following options:

> Each file should be named like ``<ISSUE_OR_PR>.<TYPE>.rst``,
> where ``<ISSUE_OR_PR>`` is an issue or a pull request number -
> whatever is most useful to link to,
> and ``<TYPE>`` is one of:
> 
> * ``feature``: New user-facing feature.
> * ``bugfix``: A bug fix.
> * ``doc``: Documentation improvement.
> * ``removal``: Deprecation or removal of public API or behavior.
> * ``change``: A change in public API or behavior.


What would you suggest? Bugfix?
I had the impression change would fit best.

Reply:
I think I'd argue the previous behavior was a bug, as it raised a nonsense exception - so yes, `bugfix`.

Reply:
Addressed in Fixup: 53c100b60437a0c55b7deae65e72c5e9c61707f2